core/src/main/java/org/opensearch/sql/expression/DSL.java

```diff
@@ -654,4 +654,9 @@ public FunctionExpression match(Expression... args) {
     return (FunctionExpression) repository
         .compile(BuiltinFunctionName.MATCH.getName(), Arrays.asList(args.clone()));
   }
+
+  public FunctionExpression match_phrase(Expression... args) {
```

Comment:
Should be camelCase here?

Reply:
This block of code copied from `MatchQuery` implementation as is. Perhaps, we need to do code clean up in scope of another task.